### PR TITLE
Fix avatar with special chars

### PR DESCRIPTION
**Pull request by chrysanthos**

### CODE REVIEW

File: app/Models/Pharmacy.php [hunk 1]
```diff
@@ -71,7 +72,17 @@ public function getSeoUrlAttribute()
...
 
-    public function getSlugOptions() : SlugOptions
+    public function getAvatarUrlAttribute()
+    {
+        $name = preg_replace("/[^a-zA-Z\p{Greek} ]+/u", "", $this->name);
+        $name = collect(explode(' ', $name))->filter()->values()->map(function ($part) {
+            return Str::substr($part, 0, 1);
+        })->take(2)->implode('');
+
+        return "https://ui-avatars.com/api/?name={$name}&color=7F9CF5&background=EBF4FF";
```
Comment (gregkos) [inline app/Models/Pharmacy.php:82]: Magic numbers :(

File: tests/Unit/AvatarUrlTest.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            ['Test Test', 'TT'],
+            ['Test A', 'TA'],
+            ['Test - Test', 'TT'],
+            ['Test A', 'TA'],
+            ['Foo Bar (Test)', 'FB'],
+            ['A', 'A'],
+            ['Ανδρέας Ανδρέου', 'ΑΑ'],
+            ['Ανδρέας (Ανδρέου)', 'ΑΑ'],
+            ['Ανδρέας - (Ανδρέου) Χρίστου', 'ΑΑ'],
+            ['Γιάννης (Κάτι) Χρίστου', 'ΓΚ'],
```
Comment (gregkos) [inline tests/Unit/AvatarUrlTest.php:33]: Why not FL from First Middle Last?

**Review by gregkos — APPROVED** [review]

File: tests/Unit/AvatarUrlTest.php (hunk 2 above)
Comment (chrysanthos) [reply tests/Unit/AvatarUrlTest.php:33]: I believe ui-avatars only supports two letters

File: app/Models/Pharmacy.php (hunk 1 above)
Comment (chrysanthos) [reply app/Models/Pharmacy.php:82]: Couldn't find a better way, ideas?

Comment (gregkos) [reply app/Models/Pharmacy.php:82]: Ideally env/config values, but at the very least a class constant IMO

File: tests/Unit/AvatarUrlTest.php (hunk 2 above)
Comment (gregkos) [reply tests/Unit/AvatarUrlTest.php:33]: Sure, but why the middle name and not first and last?

Comment (chrysanthos) [reply tests/Unit/AvatarUrlTest.php]: That made sense, but while implementing the change I realised that It's not always first middle last names 😢 
See https://github.com/drebsoft/farmakeia.com.cy/pull/64/files#diff-5360a0a216c75bd5173ff093bef6760230c11117f73586f637468f66b998eff6R36

Comment (chrysanthos) [reply tests/Unit/AvatarUrlTest.php]: But from a quick check it seems First Middle Last is the most common case so let's go with what you said.

**Review by gregkos — APPROVED** [review]

**Pull request merged**